src/render.ts

```diff
@@ -32,8 +32,14 @@ export type RenderArguments = GlobalArguments & Arguments & {
 
 export async function renderMain(argv: RenderArguments): Promise<number> {
   await configureAWS(argv);
-  const rootDocLocation = pathmod.resolve(argv.template);
-  const content = fs.readFileSync(rootDocLocation);
+
+  // Read from STDIN (0) if the template is `-`
+  // For some reason, yargs converts the `-` to `true`
+  const isStdin = typeof argv.template === 'boolean' && argv.template === true;
+  const rootDocLocation = pathmod.resolve(isStdin ? '-' : argv.template);
```

Comment:
How about `/dev/stdin` here instead of `-`? That would simplify the next line.

Reply:
Good suggestion - I'll take a look at that.

I think part of the problem is that `rootDocLocation` appears to also be used to determine relative imports in the preprocessor. Those types of things get a little weird when the template comes from STDIN, but I would expect iidy to use the current working directory for those. I dig in to see how we can clean up both use cases. I'll see how this affects error messages as well, as having `/dev/stdin` in them may be confusing.

Reply:
cwd makes most sense there. I think we should set that explicitly somehow.

Reply:
So I dug in and `rootDocLocation` is passed to `transform` and used both for debugging/error messages and resolving relative file imports. Using `$(pwd)/-` feels pretty bad (which is what the PR currently does), but splitting the import dir from the filename would require a lot of changes to the `preprocess` module. What do you think?

Reply:
I think sticking with `$(pwd)/-` is ok for now.